staff/lab/ocf-tv

```diff
@@ -70,21 +78,26 @@ def connect(args):
 if __name__ == '__main__':
     commands = {
         'connect': connect,
-        'volume': volume
+        'volume': volume,
+        'mute': volume,
     }
     parser = argparse.ArgumentParser(
         description='Control the OCF television.',
     )
     parser.add_argument('-H', '--host', type=str, default='tv')
 
-    command_subparser = parser.add_subparsers(dest='command', help='command to run')
+    command_subparser = parser.add_subparsers(dest='command')
 
     subparser_connect = command_subparser.add_parser('connect', help='open a vnc instance to view the TV screen')
 
-    subparser_volume = command_subparser.add_parser('volume', help='set the volume for a PulseAudio sink on the TV')
-    subparser_volume.add_argument('--sink', '-s', required=False, type=str, default='0')
-    subparser_volume.add_argument('amount', type=volume_in_range,
-                                  default='100', help='volume in percent between 0% and 150%')
+    subparser_volume = command_subparser.add_parser('volume', help='set the volume on the TV using PulseAudio')
+    subparser_volume.add_argument('amount', type=volume_in_range, nargs='?',
+                                  help='volume in percent between 0% and 150%.'
+                                       '+/- a relative value is also allowed.')
+
+    subparser_mute = command_subparser.add_parser('mute', help='mute the TV')
+    # dirty hack so volume() can be reused
+    subparser_mute.add_argument('--amount', default=0, help=argparse.SUPPRESS)
```

Comment:
Make the volume function take explicit arguments instead of the argparse return value.